packages/office-ui-fabric-react/src/components/Dialog/DialogContent.types.ts

```diff
@@ -42,6 +42,11 @@ export interface IDialogContentProps extends React.ClassAttributes<DialogContent
    */
   showCloseButton?: boolean;
 
+  /**
+   * Add tooltip for 'x' close button in the upper-right corner
+   */
+  closeButtonToolTip?: string;
```

Comment:
This is likely to change the API and the snapshots, which need to be updated for the build to pass.

Are you able to run these in the `packages/office-ui-fabric-react` directory?

```sh
yarn update-api
yarn update-snapshots
```

Reply:
Instead of adding a new prop, we should probably just use the existing `closeButtonAriaLabel` for the title as well. Not completely ideal but other components (Panel, TeachingBubbleContent) already do this.

Reply:
updated as you suggested using closeButtonAriaLabel as title